code/modules/antagonists/fugitive/old_god.dm

```diff
@@ -164,3 +167,31 @@
 		return TRUE
 	to_chat(owner, "<span class='warning'>Something horrible just happened to your target!</span>")
 	return FALSE
+
+
+/datum/action/cooldown/yalp_heal
+	name = "Purification"
+	desc = "Heals all followers a bit."
+	icon_icon = 'icons/mob/actions/actions_animal.dmi'
+	background_icon_state = "bg_spell"
+	button_icon_state = "god_heal"
+	cooldown_time = 600
+
+/datum/action/cooldown/yalp_heal/Trigger()
+	var/list/faithful = list()
+	var/heal_amount = -20
```

Comment:
no not here. On the /datum/action/cooldown/yalp_heal

Reply:
Maybe make it a non-negative number and then put a "-" in the adjustBruteLoss() and others